addon/templates/components/dashboard-calendar.hbs

```diff
@@ -169,5 +169,6 @@
     changeView=changeView
     selectEvent=selectEvent
     icsFeedUrl=(get (await currentUser.model) 'absoluteIcsUri')
+    icsInstructions=icsInstructionsTranslation
```

Comment:
Not going to hold this up over this, but we should move away from this pattern and instead do:
`icsInstructions=(t 'icsInstructions')` to pass in the translated text or in this case just do the translation inside of the `ics-feed` component not that the translations exist in the same repo as the component.